pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Managed Kubernetes Service Audit Logs Forwarding
+excerpt: 'Find out how to forward Audit logs from an OVH Managed Kubernetes Service cluster to Logs Data Platform'
+updated: 2024-30-01
+---
+
+ 
+## Objective
+In this guide, you will learn how to enable the forwarding of audit logs from your OVHcloud Managed Kubernetes Service (MKS) cluster to Logs Data Platform (LDP), a platform that helps you store, archive, query and visualize your logs.
+If you want to discover Logs Data Platform before reading this guide, please refer to [Log Data Platform introduction guide](https://help.ovhcloud.com/csm/en-gb-logs-data-platform-introduction?id=kb_article_view&sysparm_article=KB0058312).
+
+## Glossary
+- **Logs Data Platform:** a fully managed and secured log management platform by OVHcloud. Find more information on the [Logs Data Platform service page](https://www.ovhcloud.com/en-ie/logs-data-platform/).
+- **Data Stream:** a logical partition of logs that you create in an LDP account that you will use when ingesting, visualizing or querying your logs. Multiple sources can be stored in the same data stream, and it is the unit for defining a logs pipeline (retention policy, archiving, live streaming...), access rights and alert policies.
+- **Logs forwarding:** feature integrated in an OVHcloud product to ingest logs from its services to a *Data Stream* of an LDP account in the same OVHcloud account. The feature has to be enabled by the customer and per service.
+- **Logs forwarding Subscription:** when enabling the logs forwarding for a given OVHcloud service to a given LDP *Data Stream*, a *Subscription* is created and attached to the *Data Stream* for further management by the customer.
+- **Request Stage and Audit Level:**  audit records begin their lifecycle inside the kube-apiserver component. Each request on each stage of its execution generates an audit event, which is then pre-processed according to the policy defined by the OVHcloud Managed Kubernetes Service. This policy defines which audit events as well as which audit level (meaning which events data) are forwarded through the Audit logs of your Kubernetes cluster. For more details, refer to the Kubernetes Auditing documentation.  
+
+## Requirements
+To follow this guide, you will need:
+-	a Logs Data Platform (LDP) account with at least one active *Stream* configured. This guide will walk you through all the necessary steps.
+-	an up-and-running Managed Kubernetes Service (MKS) cluster. This guide will walk you through the steps as well.
+Both LDP account and MKS cluster must belong to the same OVHcloud accounts
```

Comment:
Not sure for this one, but maybe `account` instead of `accounts`

Reply:
```suggestion
Both LDP account and MKS cluster must belong to the same OVHcloud account.
```